ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -221,8 +225,10 @@ def fetch_tables(
                     fullyQualifiedName=fqn,
                     columns=table_columns,
                 )
+                if self.table_constraints:
+                    table_entity.tableConstraints = self.table_constraints
                 try:
-                    if self.sql_config.generate_sample_data:
+                    if not self.sql_config.generate_sample_data:
```

Comment:
@ayush-shah any reason for "not" here , this means users need to mark it as false to get the sample data